### ISSUE TITLE
whats the best way to limit the number of toasts to show?

### ISSUE THREAD

**Issue by ron51** [jirikavi/AngularJS-Toaster#134]
If I have 10 toasts and have to show only 5 at a time or 2 at a time, what's the best way to handle this situation? Can we do this with the current library?

**Comment by Stabzs** [comment]
It depends on what you mean by "5 at a time".  If you mean, always limit the number of toasts shown regardless of the number fired, yes, the 'limit' config option will do that.  if you mean queue up the remaining messages and then display them after the original set times out, no, I don't believe that is implemented.  Although if you simply wanted to do that, you could implement your own queue structure wrapped in $timeout that would simply re-pop the next potential toasts.  This might be better than adding an awkward configuration to the library since it makes more sense to keep control of invocation outside of the library.

**Comment by ron51** [comment]
I am looking to have a toaster queue as you mentioned in the second option. Instead of $timeout we can even listen to toastr.options.onHidden = function() { console.log('goodbye'); }  callback function and then handle the display toasts accordingly.

Thanks for the help!!!

**Comment by Stabzs** [comment]
@ron51 This has been merged into the build.  Can you please verify the fix and close if resolved?

**Comment by Nandabasu** [comment]
Hi,
I am using angular toaster to show warnings and errors. How can limit the toast messages to only "1". 
Below is code 

I am using toaster.min.js plugin 
toaster.pop('', '', 'Select an answer');

Where and how can I limit toasters.
Any help will be appreciated 

Thanks to all

**Comment by Dev-Pawar** [comment]
I am using angular 6 and Materilize css. I have created toast service and Injected in Interceptor to handle handle every request. 

problem is when I request one page this page have more than 2 http requests. But for every request I have some condition so that request will not be hit to server and show error message of toaster. 
**But for every wrong request toast shows message.** I want toast pop up only once. how Can i do?
